web/packages/teleterm/src/services/tshd/errors.ts

```diff
@@ -16,14 +16,26 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-export function isAccessDeniedError(error: Error): boolean {
-  return (error.message as string)?.includes('access denied');
+import { isTshdRpcError } from './cloneableClient';
+
+export function isAccessDeniedError(error: unknown): boolean {
```

Comment:
We should probably deprecate all functions in this file and make callsites use something like `isTshdRpcError(error, 'NOT_FOUND')`, where the second argument belongs to a discriminated union.

But this is not a PR which addresses the errors, we're focused on the gRPC client instead, right?

Reply:
>isTshdRpcError(error, 'NOT_FOUND')

That's a nice idea, I didn't think of it. But in general, I wanted to limit the number of changes and only updated the implementation here.

We can switch to `isTshdRpcError(error, 'NOT_FOUND')` in a separate PR.